### PR TITLE
Add Antora docs

### PR DESCRIPTION
**Pull request by lfrancke**

### CODE REVIEW

File: docs/antora.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: opa
+version: master
```
Comment (maltesander) [inline docs/antora.yml:2]: If it refers to the branch, it should be "main". I checked some other operators and it is "master" everywhere, so if im wrong just ignore this.

File: README.adoc [hunk 2]
```diff
@@ -1,10 +1,7 @@
-= Stackable Open Policy Agent Operator
+= Stackable Operator for OpenPolicyAgent
 
-This Kubernetes Operator deploys and manages the Open Policy Agent (OPA).
+This is a Kubernetes Operator to manage OpenPolicyAgent servers.
```
Comment (maltesander) [inline README.adoc:3]: I think we should use the shortcut OPA somewhere on top. I think many people can relate better to the shortcut (at least in my case).

**Review by maltesander — APPROVED** [review]
Just two small comments, you can ignore them, so approved.

File: docs/antora.yml (hunk 1 above)
Comment (lfrancke) [reply docs/antora.yml:2]: Yeah, `master` is not the branch but an Antora specific thing.

Comment (maltesander) [reply docs/antora.yml:2]: Ok did not know that ;)

File: README.adoc (hunk 2 above)
Comment (lfrancke) [reply README.adoc]: I've changed it

**Review by maltesander — APPROVED** [review]

**Pull request merged**